payment-platform-adapter/src/helpers/logger.test.ts

```diff
@@ -34,6 +34,7 @@ describe('Logger Service', () => {
 
   describe('Log', () => {
     it('should log a message with defined operation', async () => {
+      LoggerService.isDebugging = true;
```

Comment:
https://jestjs.io/docs/environment-variables
`NODE_ENV = 'test`
I think we had this configuration one of the template or d.aggregation service. We don't need to hardcode them.

Reply:
it is configurable and being used, but in the unit test it doesn't load the .env file, so I had to do that to test all code paths. Is that fine?

Reply:
I made a code suggestion. I think if we add 'test', we can remove these lines. (LoggerService.isDebugging = true)